packages/components/tabs/src/tab.spec.ts

```diff
@@ -111,46 +107,12 @@ describe('sl-tab', () => {
       expect(link).to.have.property('href', 'javascript:void(0)');
     });
 
-    it('should activate the link when pressing Enter', async () => {
-      const onClick = spy(link, 'click');
-
-      tab.focus();
-      await sendKeys({ press: 'Enter' });
-
-      expect(onClick).to.have.been.calledOnce;
-    });
-
-    it('should not activate the link when disabled and pressing Enter', async () => {
-      tab.disabled = true;
-      await el.updateComplete;
-
-      const onClick = spy(link, 'click');
-
-      tab.focus();
-      await sendKeys({ press: 'Enter' });
-
-      expect(onClick).not.to.have.been.called;
-    });
-
-    it('should activate the link when pressing Space', async () => {
```

Comment:
I think we need to have this test, because right now it's not working when we press `Space` or `Enter`, the link is not being activated in the current version.